### PR TITLE
[ACS-8045] update chrome driver link

### PR DESCRIPTION
**Pull request by akashrathod28**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x") update chrome driver to latest

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe: update chrome driver to latest

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**: https://hyland.atlassian.net/browse/ACS-8045

### CODE REVIEW

File: .github/actions/setup-chrome/action.yml [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
   steps:
       - name: Install google chrome
         shell: bash
-        run: |
-            wget -q https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_114.0.5735.133-1_amd64.deb
-            sudo apt install -y --allow-downgrades ./google-chrome-stable_114.0.5735.133-1_amd64.deb
-            sudo ln -s /usr/bin/google-chrome /usr/bin/chrome
-            chrome --version
\ No newline at end of file
+        run: './scripts/webdriver-update-newest/update-to-newest-webdriver.sh'
```
Comment (rafalszmit) [inline .github/actions/setup-chrome/action.yml:9]: these are different things, then wget was downloading browser and installing it, the script you are adding is installing web driver, that should be done after the the browser is installed. Not sure if chrome itself is already installed

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "04": "echo -------------------------------------------- Demo Shell -----------------------------------------------",
     "start": "nx serve demoshell --open",
     "start:prod": "nx serve demoshell --configuration production --open",
+    "update-webdriver": "node runany.js update-webdriver",
```
Comment (rafalszmit) [inline package.json:21]: there is no `runany.js` file in ADF

File: .github/actions/setup-chrome/action.yml (hunk 1 above)
Comment (rafalszmit) [reply .github/actions/setup-chrome/action.yml:9]: change this back to stable https://github.com/Alfresco/alfresco-ng2-components/pull/8770/files and update web driver in a different place

File: package.json (hunk 2 above)
Comment (akashrathod28) [reply package.json:21]: this I added just to run e2e as, it was not able to run due no affected changes dictated.  I will remove this. 
here we can see e2e test able to run successfully. https://github.com/Alfresco/alfresco-ng2-components/actions/runs/9253234281/job/25453146522

File: .github/actions/setup-chrome/action.yml (hunk 1 above)
Comment (akashrathod28) [reply .github/actions/setup-chrome/action.yml:9]: here we don't  need to install chrome , we got chrome version in this run
 https://github.com/Alfresco/alfresco-ng2-components/actions/runs/9253234281/job/25453146522
 
 Can we rename this setup-chrome job to update-chrome-driver to update latest driver.

**Review by rafalszmit — APPROVED** [review]
We discussed solutions on slack. I am fine with current one.

**Comment by akashrathod28** [comment]
updated with other pr https://github.com/Alfresco/alfresco-ng2-components/pull/9749/files

**Pull request closed without merging**